### PR TITLE
Feature get courses swen900132021 ag 94

### PR DESCRIPTION
**Pull request by 303221459**
Modified loadcourse() to make it return the right formatted JSON string to the front-end.
Added the response status as "200, OK".

### CODE REVIEW

**Comment by Ev-anz** [comment]
I would suggest not uploading anything related to IDE you're using locally, like the .idea folder.

File: src/courses/course1/page1.js [hunk 1]
```diff
@@ -20,5 +20,6 @@ module.exports = {
       check: (inputs) => inputs.id === 'red',
     },
   ],
+  previous: null,
```
Comment (Ev-anz) [inline src/courses/course1/page1.js:23]: Perhaps we'll leave the previous/next checks to the frontend, as according to the API these previous/next pages are not mandatory, and there's no previous page for the starting page, as well as there's no next page for the last page

**Review by Ev-anz — COMMENTED** [review]
I would suggest not uploading anything related to IDE you're using locally, like the .idea folder;

File: src/courses/course1/page1.js (hunk 1 above)
Comment (303221459) [reply src/courses/course1/page1.js:23]: ok

**Pull request closed without merging**